terraform/account/account_migrate.tf

```diff
@@ -0,0 +1,39 @@
+
+resource "aws_kms_key" "rds_migration" {
+  description = "RDS account migration"
+  policy                  = data.aws_iam_policy_document.external_cmk_access.json
+}
+
+
+resource "aws_kms_alias" "rds_migration" {
+  name          = "alias/rds_account_migrations"
+  target_key_id = "${aws_kms_key.rds_migration.key_id}"
```

Comment:
don't need the `"${}"` here as we're not interpolating a string (tf 12 syntax no longer requires it).